lib/dmv_facility.rb

```diff
@@ -0,0 +1,28 @@
+class DmvFacility
+  def create_facilities(databases)
+    datum = databases.map do |data|
+      Facility.new({
+        name: access_name(data),
+        address: access_address(data),
+        phone: access_phone(data),
+      })
+    end
+  end
+
+  # require 'pry'; binding.pry
+  def access_name(datum)
+    datum[:title]
+  end
+
+  def access_address(datum)
+    datum.include?("oregon")
```

Comment:
By having helper methods for each attribute, you set yourself up nicely for extending things to different states